src/convertToObject.js

```diff
@@ -12,7 +12,26 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const objectCss = {};
+  let sourceStringSplit = sourceString.split(';');
+
+  sourceStringSplit = sourceStringSplit.map((element) => {
+    return element.trim();
+  });
+
+  sourceStringSplit = sourceStringSplit.filter((element) => {
+    if (element !== '') {
+      return element.trim();
+    }
+  });
+
+  for (let i = 0; i < sourceStringSplit.length; i++) {
+    const newsourceString = sourceStringSplit[i].split(': ');
```

Comment:
```suggestion
    const newsourceString = sourceStringSplit[i].split(':');
```
краще без пробілу, ти там і так через trim обрізаєш зайве